src/server/api-router.ts

```diff
@@ -67,7 +64,7 @@ apiRouter.post("/login", async (req, res) => {
       identity: {
         id: token.uid,
         uid: token.uid,
-        uidc: token.uid.slice(-6) + kratosSession.identity.id.slice(-6), // FIXME: Get from backend
+        uidc: token.uid.slice(-6) + token.uid.slice(-6), // FIXME: Get from backend
```

Comment:
Why wouldn't we use the last 12 chars?

Reply:
it doesn't matter here really, the uidc is not used in bbw yet, I made a temp value here for consistency.